weather_portal/settings.py

```diff
@@ -40,6 +40,7 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
+    "registration_login"
```

Comment:
Przy takiej liście pionowej zawsze dajemy przecinek po ostatnim elemencie
```suggestion
    "registration_login",
```